README.md

```diff
@@ -94,64 +94,14 @@ twig:
 #### Required
 
 First, you have to select your language, other settings are optional (see below).
+If you don't have this file you can create one.
 
 ```yaml
-# config/packages/kms_froala_editor.yaml
+# config\packages\config.yaml 
 kms_froala_editor:
     language: 'nl'
 ```
 
-#### Other options
```

Comment:
Why do you want to move it? It's about bundle configuration

Reply:
@jmsche added this at end , as in all other frameworks "Options Configuration" was passed at end of setup steps. So removed from here and added at end.